platforms/core-runtime/build-configuration/src/integTest/groovy/org/gradle/interal/buildconfiguration/tasks/UpdateDaemonJvmIntegrationTest.groovy

```diff
@@ -286,4 +310,109 @@ class UpdateDaemonJvmIntegrationTest extends AbstractIntegrationSpec implements
             ["WINDOWS", "AARCH64"]: "https://server?platform=WINDOWS.AARCH64&toolchain=20.FOO",
         ])
     }
+
+    def "resolvers failing to return any download URLs is considered an error"() {
+        given:
+        writeJvmCriteria(Jvm.current())
+        settingsFile << applyToolchainResolverPlugin("CustomToolchainResolver", noUrlResolverCode())
+
+        when:
+        fails "updateDaemonJvm", "--jvm-version=20", "--jvm-vendor=FOO"
+
+        then:
+        // TODO The description is different with CC on
+//        failureDescriptionContains("Execution failed for task ':updateDaemonJvm'")
+        failureHasCause("Toolchain resolvers did not return download URLs providing a JDK matching {languageVersion=20, vendor=vendor matching('FOO'), implementation=vendor-specific} for any of the requested platforms")
```

Comment:
Since this is a new message, the IDE should incorporate this on the error handling for the Daemon toolchain to offer quick fixes. I'll create a task to do it: 

<img width="1533" alt="Screenshot 2025-02-05 at 13 11 38" src="https://github.com/user-attachments/assets/9a96c4b8-6b08-467b-9d10-aa1ce2dc6408" />

Reply:
Sounds good, but let's make sure to use the final message - aka this is not final yet.